### PR TITLE
Nearone mocknet

### PR DESCRIPTION
**Pull request by andrei-near**
Moving mocknet network and tests to N1 infra

### CODE REVIEW

**Comment by VanBarbascu** [comment]
If we land this now, we will block the existing mocknets. One way to go here is to look for an env var and if it is not set, default it to `nearone-mocknet`.

File: pytest/lib/mocknet.py [hunk 1]
```diff
@@ -25,7 +25,7 @@
 NODE_SSH_KEY_PATH = None
 NODE_USERNAME = 'ubuntu'
 NUM_ACCOUNTS = 26 * 2
-PROJECT = 'near-mocknet'
+PROJECT = 'nearone-mocknet'
```
Comment (VanBarbascu) [inline pytest/lib/mocknet.py:28]:
```suggestion
import os
PROJECT = os.getenv('MOCKNET_PROJECT', 'nearone-mocknet')
```

**Comment by andrei-near** [comment]
Agree @VanBarbascu . updated

**Review by VanBarbascu — APPROVED** [review]
Looks good!

**Pull request merged**